docs/user/azure/credentials.md

```diff
@@ -0,0 +1,32 @@
+# Service Principal
+
+Before proceeding with the OpenShift install, you should create a service principal with administrative rights for your subscription following the steps
+outlined here:
+
+[Azure: Creating an Service Principal][sp-create]
+
+## Step 1: Create a Service Principal
+
+You can create a Service Principal using Azure [portal][sp-create-portal] or Azure [cli][sp-create-cli]
+
+## Step 2: Attach Administrative Role
+
+Azure installer creates new identities for the cluster and therefore requires access to create new roles, and role assignments. Therefore, you will require the service
+principal to have at least `Contributor` and `User Access Administrator` [roles][built-in-roles] assigned in your subscription.
+
+You can create role assignments for your service principal using [portal][sp-assign-portal] or Azure [cli][sp-assign-cli]
```

Comment:
Their docs have "[using the Azure portal][1]" and "[The Azure CLI][2], so we probably want `... using [the portal][sp-assign-portal] or [CLI][sp-assign-cli].`.

[1]: https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-create-service-principals#create-service-principal-for-azure-ad
[2]: https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest#manage-service-principal-roles

Reply:
https://github.com/openshift/installer/compare/fc6532c8637a2c37f4c0a028da1a5a430ac19302...3bb0cfefaf513837f94ff94c1bc59e047421323b